langchain/scripts/create-entrypoints.js

```diff
@@ -257,6 +257,8 @@ const entrypoints = {
   "storage/in_memory": "storage/in_memory",
   "storage/ioredis": "storage/ioredis",
   "storage/vercel_kv": "storage/vercel_kv",
+  "storage/upstash_redis": "storage/upstash_redis",
```

Comment:
Also needs to go under `requiresOptionalDependency` below since it uses `@upstash/redis`